File: day 40/hw/hw.py
def hello_world():
    print("Hello, World!")


def add_numbers(num1, num2):
    print(num1 + num2)


def multiply_by_ten(number):
    return number * 10


def greet(name="Guest"):
    print(f"Hello, {name}!")


def outer_function():
    def inner_function():
        print("I am the inner function!")
    inner_function()


def check_even_odd(numbers):
    for number in numbers:
        if number % 2 == 0:
            print("Even")
        else:
            print("Odd")

File: day 71/hw/hw.js
let numbers = [10, 21, 32, 43, 54, 65];

console.log("All elements:");
for (let i = 0; i < numbers.length; i++) {
    console.log(numbers[i]);
}

console.log("Index and elements:");
for (let i = 0; i < numbers.length; i++) {
    console.log("Index " + i + " -> " + numbers[i]);
}

console.log("Even numbers:");
for (let i = 0; i < numbers.length; i++) {
    if (numbers[i] % 2 === 0) {
        console.log(numbers[i]);
    }
}

console.log("Odd numbers:");
for (let i = 0; i < numbers.length; i++) {
    if (numbers[i] % 2 !== 0) {
        console.log(numbers[i]);
    }
}

let sum = 0;
for (let i = 0; i < numbers.length; i++) {
    sum += numbers[i];
}
